### PR TITLE
Updates to Font Awesome 4.4.0

### LINKED ISSUE #16: update to 4.4.0
There are new icons that aren't appearing.

### PR DESCRIPTION
**Pull request by hello-jason**
Fixes #16

### CODE REVIEW

File: app/assets/stylesheets/font-awesome/_animated.scss [hunk 1]
```diff
@@ -1,36 +1,34 @@
-@if $fa-enable-utilities {
```
Comment (cristianferrarig) [inline app/assets/stylesheets/font-awesome/_animated.scss]: Hi @hello-jason Thanks for your contribution :)
Please keep the if statement in order to don't break the $fa-enable-utilities configuration option to not include utilities classes.

**Comment by hello-jason** [comment]
Should be good now.

**Comment by douglascamata** [comment]
ping

**Comment by cristianferrarig** [comment]
Sorry for the delay @hello-jason and thanks! a lot!
And thanks for the ping @douglascamata :D

The gem is ready to use.

**Comment by douglascamata** [comment]
@cristianferrarig thank you for being so fast! :+1:

**Pull request merged**